htdocs/update.bml

```diff
@@ -663,7 +662,11 @@ head<=
     //]]>
     </script>^;
 
-    $ret .= (! $LJ::REQ_HEAD_HAS{'chalresp_js'}++) ? $LJ::COMMON_CODE{'chalresp_js'} : "";
+    my $chalresp_js = qq{
+<script type="text/javascript" src="$LJ::JSPREFIX/md5.js"></script>
+};
+
+    $ret .= (! $LJ::REQ_HEAD_HAS{'chalresp_js'}++) ? $chalresp_js : "";
```

Comment:
This one, otoh, MIGHT be used for crossposting. I don't understand that flow well enough to be sure. 

Sidenote, this "check for zeroness while mutating as a side effect, also you're inside a ternary" thing is... sure something.